python-flask-audit-logs-example/README.md

```diff
@@ -0,0 +1,127 @@
+
+   - Your [WorkOS API key](https://dashboard.workos.com/api-keys)
+   - Your [SSO-specific, WorkOS Client ID](https://dashboard.workos.com/configuration)
+
+6. Ensure you're in the root directory for the example app, `python-flask-sso-example/`. Create a `.env` file to securely store the environment variables. Open this file with the Nano text editor. (This file is listed in this repo's `.gitignore` file, so your sensitive information will not be checked into version control.)
+
+   ```bash
+   (env) $ touch .env
+   (env) $ nano .env
+   ```
+
+7. Once the Nano text editor opens, you can directly edit the `.env` file by listing the environment variables:
+
+   ```bash
+   WORKOS_API_KEY=<value found in step 6>
+   WORKOS_CLIENT_ID=<value found in step 6>
+   APP_SECRET_KEY=<any string value you\'d like>
+   ```
+
+   To exit the Nano text editor, type `CTRL + x`. When prompted to "Save modified buffer", type `Y`, then press the `Enter` or `Return` key.
+
+8. Source the environment variables so they are accessible to the operating system.
+
+   ```bash
+   (env) $ source .env
+   ```
+
+   You can ensure the environment variables were set correctly by running the following commands. The output should match the corresponding values.
+
+   ```bash
+   (env) $ echo $WORKOS_API_KEY
+   (env) $ echo $WORKOS_CLIENT_ID
+   ```
+
+9. In `python-flask-sso-example/app.py` change the `CUSTOMER_CONNECTION_ID` string value to the connection you will be testing the login for. This can be found in your WorkOS Dashboard.
+
+10. The final setup step is to start the server.
+
+```bash
+(env) $ flask run
+```
+
+If you are using macOS Monterey and port 5000 is not available and you'll need to start the app on a different port with this slightly different command.
+
+```bash
+(env) $ flask run -p 5001
+```
+
+You'll know the server is running when you see no errors in the CLI, and output similar to the following is displayed:
+
+```bash
+* Tip: There are .env or .flaskenv files present. Do "pip install python-dotenv" to use them.
+* Environment: production
+WARNING: This is a development server. Do not use it in a production deployment.
+Use a production WSGI server instead.
+* Debug mode: off
+* Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
+```
+
+Navigate to `localhost:5000`, or `localhost:5001` depending on which port you launched the server, in your web browser. You should see the WorkOS logo and a place to enter your Organization ID. The next steps occur in your WorkOS dashboard.
+
+You can stop the local Flask server for now by entering `CTRL + c` on the command line.
+
+## Audit Logs Setup with WorkOS
+
+11. Follow the [Audit Logs configuration steps](https://workos.com/docs/audit-logs/emit-an-audit-log-event/sign-in-to-your-workos-dashboard-account-and-configure-audit-log-event-schemas) to set up the following 5 events that are sent with this example:
+
+1. Action title: "user.signed_in" | Target type: "team"
```

Comment:
not sure why but all the steps after this are just "1" - we can probably update that